tests/system/data_sources/test_postgres.py

```diff
@@ -460,3 +463,87 @@ def test_schema_validation(cloud_sql):
 
     for validation in df.to_dict(orient="records"):
         assert validation["validation_status"] == consts.VALIDATION_STATUS_SUCCESS
+
+
+@mock.patch(
+    "data_validation.state_manager.StateManager.get_connection_config",
+    return_value=CONN,
+)
+def test_schema_validation_core_types(mock_conn):
+    parser = cli_tools.configure_arg_parser()
+    args = parser.parse_args(
+        [
+            "validate",
+            "schema",
+            "-sc=mock-conn",
+            "-tc=mock-conn",
+            "-tbls=pso_data_validator.dvt_core_types",
+            "--filter-status=fail",
+        ]
+    )
+    config_managers = main.build_config_managers_from_args(args)
+    assert len(config_managers) == 1
+    config_manager = config_managers[0]
+    validator = data_validation.DataValidation(config_manager.config, verbose=False)
+    df = validator.execute()
+    # With filter on failures the data frame should be empty
+    assert len(df) == 0
+
+
+@mock.patch(
+    "data_validation.state_manager.StateManager.get_connection_config",
+    return_value=CONN,
+)
+def test_column_validation_core_types(mock_conn):
+    parser = cli_tools.configure_arg_parser()
+    # TODO Add col_datetime,col_tstz to --sum string below when issue-XXX is complete. Or change whole string to * if YYY is also complete.
```

Comment:
Was an issue opened for these Postgres bugs?

Reply:
No, I've made a mistake there, PostgreSQL tests can run with these set to `*`. Thanks for spotting this.
I've modified the code and tests re-running.